modin/pandas/utils.py

```diff
@@ -105,3 +107,35 @@ def is_scalar(obj):
     from .base import BasePandasDataset
 
     return not isinstance(obj, BasePandasDataset) and pandas_is_scalar(obj)
+
+
+def from_modin_frame(df, sortorder=None, names=None):
```

Comment:
Because `from_modin_frame` doesn't belong to any class, it is not clear what return value is. So, I would suggest renaming function to `from_modin_frame_to_mi`.